src/components/organisms/SwitchToChromeModal/SwitchToChromeModal.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Modal } from "react-bootstrap";
+
+import { DOWNLOAD_CHROME_URL } from "settings";
+
+import { getExtraLinkProps } from "utils/url";
+
+import { Button } from "components/atoms/Button";
+
+import "./SwitchToChromeModal.scss";
+
+interface SwitchToChromeModalProps {
+  show: boolean;
+  onHide: () => void;
+}
+
+export const SwitchToChromeModal: React.FC<SwitchToChromeModalProps> = ({
```

Comment:
A modal popup is also a disruptive to what the users want to do, not sure if "Download Chrome" is important enough to disrupt their work.

Reply:
@goran-peoski-work user can click outside its borders or I can add the X / button like "I want to use my browser" and it'll close

Reply:
@dmitrii-zolotukhin It popping up is the disruption, doesn't matter how they close it afterwards. For something that isn't core to the UX it should probably be more subtle, just sitting somewhere, visible, but not disturbing. My take on it, though. I suggest you ping the @sparkletown/product channel (on Slack) and see if they agree on what/where/how/when etc.

Reply:
@goran-peoski-work we can try to implement it through that library https://www.npmjs.com/package/react-toastify, 
in the future this can be also used to notify user about events
Or the adding of that library is an extra stuff for the project and it's better to implement it by hand?

Reply:
@dmitrii-zolotukhin  Toast or popup... why does it have to be something actively appearing? What's wrong with a text and a link on a landing page?

Reply:
On a related note, there is already an issue, worked on by @ermiaSparkle about a notification system https://github.com/sparkletown/internal-sparkle-issues/issues/1005

Reply:
@goran-peoski-work so who is working on that issue? or what? I don't understand what I should do in that case.

What is more, should I do this notification only on landing or on all pages? Make this moment clear, please

Reply:
In case it was missed, from https://github.com/sparkletown/sparkle/pull/2258#discussion_r702960342:

----
> I suggest you ping the @sparkletown/product channel (on Slack) and see if they agree on what/where/how/when etc.

---
I only provided comments to the assumptions you took about how it would work.
There is nothing clearly decided.
The final decision is up to product, once they agree on how or if this functionality is going to be done.